Dockerfile.updater-core

```diff
@@ -110,17 +110,18 @@ RUN for ecosystem in git_submodules terraform github_actions hex elm docker nuge
 
 WORKDIR $DEPENDABOT_HOME/dependabot-updater
 
+# RubyGems & Bundler should be bumped together following these steps:
+# * Bump RubyGems version below. That will also automatically update the default Bundler version.
+# * Regenerate `updater/Gemfile.lock` via `BUNDLE_GEMFILE=updater/Gemfile bundle lock --update --bundler`.
+# * Regenerate `Gemfile.lock` via `bundle lock --update --bundler`.
```

Comment:
Can we dump this in some `script/update-bundler` script? Maybe it can even pull down the latest version?

Something like this?

```bash
#!/bin/bash

# Function to get the latest stable release version from RubyGems
get_latest_release() {
    gem_name=$1
    curl -s "https://rubygems.org/api/v1/versions/$gem_name.json" | jq -r 'map(select(.prerelease == false)) | first | .number'
}

# Get the latest stable releases for Bundler and RubyGems
latest_bundler=$(get_latest_release "bundler")
latest_rubygems=$(get_latest_release "rubygems-update")

# Update RubyGems to the latest version
echo "Updating RubyGems to version $latest_rubygems..."
gem update --system $latest_rubygems

# Install the latest Bundler
echo "Installing Bundler version $latest_bundler..."
gem install bundler --version $latest_bundler

# Update the Bundler version in Gemfile.lock and updater/Gemfile.lock
BUNDLE_GEMFILE=updater/Gemfile bundle lock --update --bundler
bundle lock --update --bundler

echo "Update complete!"
```

Reply:
Great idea. If you update the Dockerfile, you'll probably want to rebuild it at least once, so I think just

```shell
# Update the Bundler version in Gemfile.lock and updater/Gemfile.lock
BUNDLE_GEMFILE=updater/Gemfile bundle lock --update --bundler
bundle lock --update --bundler

echo "Update complete!"
```

may do the trick? `bundle lock --update --bundler` does not need the latest version installed.

Reply:
Script idea makes sense to me, but I'm also 👍 on just shipping this as-is so that at least we land the improvement and then do the script as a follow-on... that way if we have to revert at all or something we still have this original improvement landed.

Reply:
@jurre , @deivid-rodriguez if we are good for this ticket I am going to ship this PR. Maybe scripting part can be handled in another PR?

Reply:
No objection from my end to merge as is

Reply:
Going to deployed it then.

Reply:
It is deployed.

Reply:
Thank you!